_posts/2022-08-11-temp-title/my-4-biggest-internship-takeaways.Rmd

```diff
@@ -0,0 +1,50 @@
+---
+title: "My 4 Biggest Internship Takeaways"
+description: |
+  What I've learned after interning at the PCCTC.
+author:
+  - name: Kirina Sirohi
+    url: {}
+base_url: https://www.pipinghotdata.com/
+date: 2022-08-19
+preview: cover_image.png
```

Comment:
What do you think about using the screenshot from our last team meeting instead for a cover photo?